Puzzle/src/org/fundacionjala/devops/bootcamp/AsisstedBoard/AiMenu.java

```diff
@@ -0,0 +1,77 @@
+package org.fundacionjala.devops.bootcamp.AsisstedBoard;
+
+
+import org.fundacionjala.devops.bootcamp.History.MovementsHistory;
+import org.fundacionjala.devops.bootcamp.Move.Movement;
+import org.fundacionjala.devops.bootcamp.NormalBoard.Board;
+
+import java.util.Scanner;
+
+public class AiMenu {
```

Comment:
Are you trying to simulate IA?

Reply:
Indeed! Lol! It's like this class & methods are taking care of helping the user wityh the assisted mode in the Puzzle game